api_demo/app.py

```diff
@@ -0,0 +1,44 @@
+import pathlib
+import aioreloader
+
+import aiopg.sa
+import yaml
+from aiohttp.web_app import Application
+
+from api_demo.routes import setup_routes
+
+BASE_DIR = pathlib.Path(__file__).parent.parent
+config_path = BASE_DIR / 'config.yaml'
+
+
+async def pg_context(app):
+    conf = app['config']['postgres']
+    engine = await aiopg.sa.create_engine(
+        database=conf['database'],
+        user=conf['user'],
+        password=conf['password'],
+        host=conf['host'],
+        port=conf['port'],
+        minsize=conf['minsize'],
+        maxsize=conf['maxsize'],
+    )
+    app['db'] = engine
+
+    yield
+
+    app['db'].close()
+    await app['db'].wait_closed()
```

Comment:
Remove please connection to a db for now) And what about config?) Everybody can see it) You mustn't store app keys in a repo